etc/api/annotate-all-existing-analyses.sh

```diff
@@ -0,0 +1,74 @@
+#! /bin/bash
+# ./annotate-all-existing-analyses.sh
+
+usage() {
+  echo " "
+  echo "usage: $0"
+  echo " "
+  echo " -h Prints usage"
+  echo " "
+  echo "Kicks off annotation jobs for all the existing analyses in Rosalution"
+  echo " "
+  echo "To run the annotations, please log in to Rosalution to retrieve the"
+  echo "Client ID and Client Secret credentials. These can be found by"
+  echo "clicking on your username or going to: <rosalution url>/rosalution/account"
+  echo " "
```

Comment:
An additional note here for the time being in the usage to indicate it may need to be run multiple times due to Rosalution annotation system not being built for large sets of annotations queued.

Reply:
Updated the help with a note to run this script more than once if needed.

```
usage() {
  echo " "
  echo "usage: $0"
  echo " "
  echo " -h Prints usage"
  echo " "
  echo "Kicks off annotation jobs for all the existing analyses in Rosalution"
  echo " "
  echo "To run the annotations, please log in to Rosalution to retrieve the"
  echo "Client ID and Client Secret credentials. These can be found by"
  echo "clicking on your username or going to: <rosalution url>/rosalution/account"
  echo " "
  echo "Note: This script may need to be run multiple times due to the Rosalution "
  echo "annotation system not being built for large sets of annotations queued."
  echo " "
  echo "Please install jq for this script to work. https://stedolan.github.io/jq/"
  echo " "
  exit
}
```